src/table.ts

```diff
@@ -795,16 +797,17 @@ class Table {
   private _mutate(
     method: string,
     rows: object | object[],
-    callback: s.CommitCallback
+    callback: CommitCallback
   ): void {
     this.database.runTransaction((err, transaction) => {
       if (err) {
         callback(err);
         return;
       }
 
-      transaction[method](this.name, rows);
-      transaction.commit(callback);
+      // tslint:disable-next-line: no-any
+      (transaction as any)![method](this.name, rows);
```

Comment:
Why do we need to cast this to `any`?

Reply:
As We don't have any key-value property in `Transaction`, it gives below error:
`No index signature with a parameter of type 'string' was found on type 'Transaction'`

Reply:
@laljikanjareeya interesting, what would happen if we changed `method` from `string` to something like

```ts
method: 'delete' | 'insert' | 'etc'
```

Reply:
Yes, It's working Thanks!